vizro-core/examples/features/app.py

```diff
@@ -331,6 +342,62 @@
     ],
 )
 
+selectors = vm.Page(
+    title="Selectors",
+    layout=vm.Layout(grid=[[0], [1], [1], [1], [2], [2], [2]], row_min_height="170px", row_gap="24px"),
+    components=[
+        vm.Card(
+            text="""
+        A selector can be used within the **Parameter** or **Filter** component to allow the user to select a value.
+
+        The following selectors are available:
+        * Dropdown (**categorical** multi and single option selector)
+        * Checklist (**categorical** multi option selector only)
+        * RadioItems (**categorical** single option selector only)
+        * RangeSlider (**numerical** multi option selector only)
+        * Slider (**numerical** single option selector only)
+
+        """
+        ),
+        vm.Table(
+            id="table-gapminder", figure=dash_data_table(data_frame=gapminder, page_size=10), title="Gapminder Data"
+        ),
+        vm.Table(id="table-tips", figure=dash_data_table(data_frame=tips, page_size=10), title="Tips Data"),
+    ],
+    controls=[
+        vm.Filter(
+            targets=["table-gapminder"],
+            column="year",
+            selector=vm.RangeSlider(title="Range Slider (Gapminder - year)"),
+        ),
+        vm.Filter(
+            targets=["table-gapminder"],
+            column="continent",
+            selector=vm.Checklist(title="Checklist (Gapminder - continent)"),
+        ),
+        vm.Filter(
+            targets=["table-gapminder"],
+            column="country",
+            selector=vm.Dropdown(title="Dropdown (Gapminder - country)"),
+        ),
+        vm.Filter(
+            targets=["table-tips"],
+            column="day",
+            selector=vm.Dropdown(title="Dropdown (Tips - day)", multi=False),
+        ),
+        vm.Filter(
+            targets=["table-tips"],
+            column="sex",
+            selector=vm.RadioItems(title="Radio Items (Tips - sex)"),
+        ),
+        vm.Filter(
+            targets=["table-tips"],
+            column="smoker",
+            selector=vm.Slider(title="Slider (Tips - smoker)", step=1, value=1),
```

Comment:
This is not the best example of a `Slider` given that it's just a binary choice. Do we have any real, continuous numerical column we could use instead here?

Reply:
Or maybe use it as a parameter? I think filtering on a single number will always be tricky, but having opacity on a graph is always a good one e.g.

In general I think having a graph on this page would be good

Reply:
@antonymilne - I find it a bit tricky because the slider will usually reduce the number of observations drastically 😄 Which is why I didn't want to use the "tips" column as if you would use that for the slider, there would be normally only one observation left. If you combine it with the categorical single-select dropdown, they usually lead up to an empty table. Hence, why I just transformed the binary column to a numeric one, as this at least left some observations for the other selectors. I can also just add another chart below with another data set and apply the Slider only there?

Reply:
Yeah, that makes sense. What about the `size` column for the tips dataset though? Does that lead to an empty table too often?

If that doesn't work well then fine to leave how you have it with a binary slider I think.

Reply:
I went for the size column - I just played around with some default value settings for the other selectors such that we have some nice data and now it works fine 👍